lib/osf-components/addon/components/file-metadata-manager/component.ts

```diff
@@ -181,14 +181,20 @@ export default class FileMetadataManagerComponent extends Component<Args> {
         this.inEditMode = true;
     }
 
+    @action
+    rollback() {
+        this.changeset.rollback();
+    }
+
```

Comment:
Why are we putting `this.changeset.rollback()` here and invoke this action in the task?

Reply:
Please see the other comment below; I have included reference to both.

Reply:
I'm not convinced this needs to be in a separate action. I think in line 197, we can just call `this.changeset.rollback()` and remove this.